DEV_NOTES.md

```diff
@@ -313,7 +313,7 @@ The Contacts Provider may consolidate multiple contacts belonging to different a
 them into a single entry in the Contacts table whilst maintaining the separate entries in the 
 RawContacts table.
 
-A more common scenario that causes multiple RawContacts per Contact is when two or more Contacts are
+A more likely scenario that causes multiple RawContacts per Contact is when two or more Contacts are
```

Comment:
Yes. I have OCD. I look for all mentions of "common" and refactored it. Even outside of code. The only mentions of the common data kinds are usages of `ContactsContract.CommonDataKinds`.